src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -233,6 +233,13 @@ protected void setHeaders(HttpUriRequest httpUriRequest) throws IOException {
         // Content type and accept header are determined by the child class
         setContentTypeHeader(httpUriRequest);
         setAcceptHeader(httpUriRequest);
+
+        // Walk through custom headers and set
+        Map<String, String> headers = this.configuration.getHeaders();
+        for (Map.Entry<String, String> header : headers.entrySet()) {
+            Header authHeader = new BasicHeader(header.getKey(), header.getValue());
```

Comment:
We're not guaranteed that this is an authHeader, so maybe a better name is `extraHeader` or something similar